pkg/types/configuration.go

```diff
@@ -49,6 +49,9 @@ type Configuration struct {
 	// Qemu protocol is 32bits big endian size of the packet, then the packet.
 	// Hyperkit protocol is handshake, then 16bits little endian size of packet, then the packet.
 	Protocol Protocol
+
+	// Address and port for ssh on virtual machine
+	SSHHostPort string
```

Comment:
I think you can remove this configuration and keep this SSHHostPort only in cmd/gvproxy. We don't need it outside of it.

Reply:
Sure will pull that out.